### PR TITLE
feat: add product tracking

### PR DESCRIPTION
**Pull request by sogunshola**

### CODE REVIEW

File: streams/functions/src/index.ts [hunk 1]
```diff
@@ -17,6 +18,7 @@ const firestore = app.firestore();
 Moralis.start({
   apiKey: config.moralisApiKey,
 });
+Moralis.Core.config.set(CoreConfig.product, 'firebase-auth');
```
Comment (b4rtaz) [inline streams/functions/src/index.ts:21]: Should be `firebase-streams`.

**Review by b4rtaz — CHANGES_REQUESTED** [review]

File: streams/functions/src/index.ts (hunk 1 above)
Comment (sogunshola) [reply streams/functions/src/index.ts]: updated

Comment (b4rtaz) [reply streams/functions/src/index.ts]: Do we have confirmed the product header with the backend team?

Comment (sogunshola) [reply streams/functions/src/index.ts]: Yes I have spoken with the api team and they are adding it to next sprint https://moralisweb3.slack.com/archives/C02J5L3PU4U/p1666702972932029

But I think we can still add the header while we wait for them.

**Pull request merged**